users/src/config/index.js

```diff
@@ -0,0 +1,20 @@
+
+require('dotenv').config();
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
require("dotenv").config();
```